lib/command.js

```diff
@@ -40,6 +42,16 @@ export const parseCommand = command => {
 	return tokens;
 };
 
+const parseExpression = expression => {
+	if (typeof expression === 'object' && expression !== null && 'stdout' in expression) {
+		return Buffer.isBuffer(expression.stdout)
+			? expression.stdout.toString()
+			: String(expression.stdout);
```

Comment:
```suggestion
			: expression.stdout;
```

If `stdout` is not `undefined` nor a buffer, it should be guaranteed to be a string.